bundles/org.openhab.binding.ipobserver/src/main/java/org/openhab/binding/ipobserver/internal/IpObserverHandler.java

```diff
@@ -333,11 +395,19 @@ private void setupChannels() {
     public void initialize() {
         config = getConfigAs(IpObserverConfiguration.class);
         updateStatus(ThingStatus.UNKNOWN);
```

Comment:
Move this updateStatus in your else branch, just before calling your scheduled task.